src/main/scala/cromwell/engine/backend/jes/authentication/JesAuthentication.scala

```diff
@@ -20,7 +34,12 @@ object ProductionJesConnection {
  */
 trait JesAuthentication { self: JesConnection =>
 
-  def authenticated[A](f: JesInterface => A) = f(jesConnection)
+  def authenticateAsCromwell[A](f: JesInterface => A) = f(jesCromwellConnection)
+
+  /**
+   * Important note: Will default back to cromwell authentication if the configuration for user authentication has not been set or if the refreshToken has been supplied.
```

Comment:
Is this saying that if a workflow is submitted with a refresh token the customer's billing project is billed, otherwise we bill ourselves?